src/Map/Map.js

```diff
@@ -118,12 +121,14 @@ class Map extends React.Component {
 
     return (
       <>
-        <StyledMap
-          id={this.target}
-          fullScreen={fullScreen}
-          style={style}>
-          <MapLogo logoPosition={logoPosition} translations={translations} />
-        </StyledMap>
+        {!this.passedMap &&
```

Comment:
why couldn't we have just done `{!this.props.map &&` here...re-assigning map to a var that we're never going to use seems odd

Reply:
great point- this can be updated to just check the prop 🤦